crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -326,76 +345,140 @@ impl UncompressedDataManager {
             .await
             .map_err(|error| error.to_string())?;
 
+        // unwrap() is safe as lock() only returns an error if the lock is poisoned.
+        self.used_multivariate_memory_metric
+            .lock()
+            .unwrap()
+            .append(-(data_points.get_array_memory_size() as isize), true);
+
         // Return the memory used by the data points to the pool right before they are de-allocated.
         self.memory_pool
-            .free_uncompressed_memory(data_points.get_array_memory_size());
+            .free_multivariate_memory(data_points.get_array_memory_size());
+
+        // Print a single warning if any buffers are spilled so ingestion can be optimized.
+        if buffers_are_spilled {
+            warn!("Forced to spill uncompressed buffers, reduce buffer size or increase memory.");
```

Comment:
detail: this could potentially be misunderstood as if MDB was forced to reduce buffer size or increase memory. Perhaps replace comma with full stop or colon

Reply:
Fixed.